public/javascript/actions.js

```diff
@@ -0,0 +1,14 @@
+function deleteReview(id) {
+
+	var review = document.getElementById(id);
+	var myRequest = new XMLHttpRequest();
+	myRequest.onreadystatechange = function(id) {
+		if(myRequest.readyState == 4 && myRequest.status == 204) {
+			review.remove();
```

Comment:
Si tu vas utiliser cette remove, tu pars du principe que sur ta view t'as un élément (div, tr, a, n'importe lequel)
donc pour pouvoir supprimer dans to fichier jade, la review doit être entouré d'une div avec un id pour le supprimer
si tu utilises cette façon de faire, le location.reload n'est pas nécessaire.